artillery/artillery.yaml

```diff
@@ -2,7 +2,7 @@ config:
   target: 'http://localhost:3002'
   http:
     pool: 10
```

Comment:
What's "pool" for?

Reply:
Pool just limits the number of network connections virtual users can utilize. Helps conserve CPU for higher request counts.